### DISCUSSION TITLE
欢迎来到 码良的讨论区

### DISCUSSION THREAD

**Discussion by ly525** [ymm-tech/gods-pen#63] [General]
## 👋 Welcome!
大家好，欢迎大家来到 码良 社区/论坛/讨论区，我们希望让每一次交流被沉淀！
因为微信群等聊天工具有一些痛点：
 - 聊天内容有些分散，并不一定是技术 + 产品相关内容，有很多无意义内容
 - 新入群的同学无法快速融入，不了解上下文
 - 聊天内容，无法作为社区好的沉淀
 - 等等
 
因此，我们申请了 [Discussions](https://github.com/ymm-tech/gods-pen/discussions) 作为 码良 的社区/论坛/讨论区。我们希望你:

 * 问一些你想知道的问题
 * 分享你的想法
 * 与社区里其它同学接触、交流
 * 欢迎别人并且开放思想。记住这是一个我们大家共建的社区
 * 将心比心，你认真提问，大家认真解答
 * 务必阅读：[如何提问，并获得高回复率——提问的智慧](https://zhuanlan.zhihu.com/p/19779979)

## 交流边界
 * 低代码、可视化搭建相关产品、技术方向
 * 低代码、可视化搭建相关技术
 * 码良 相关技术、产品
 * 不划水、发无意义内容

**Comment by makerjackie** [comment]
很棒！总之issue负责解决bug和feature，其他的讨论都在Discussion中进行。不过由于discussion还在beta之中，还需要培养大家的使用习惯
